README.md

```diff
@@ -552,7 +552,7 @@ Additional support options are at https://wiki.lyrasis.org/display/DSPACE/Suppor
 
 DSpace also has an active service provider network. If you'd rather hire a service provider to
 install, upgrade, customize or host DSpace, then we recommend getting in touch with one of our
-[Registered Service Providers](http://www.dspace.org/service-providers).
+[Registered Service Providers](https://www.dspace.org/service-providers).
```

Comment:
Update to https://dspace.org/registered-service-providers/ please.  It appears this link is now accidentally broken.